Source/swiftlint/Commands/LintCommand.swift

```diff
@@ -16,6 +17,8 @@ struct LintCommand: CommandProtocol {
     let verb = "lint"
     let function = "Print lint warnings and errors (default command)"
 
+    private static let violationsAccumulatorQueue = DispatchQueue(label: "io.realm.swiftlint.violationsAccumulator")
```

Comment:
I know probably no one will use multiple linters at once, but shouldn't this queue be per instance?